components/resources/commands.grdp

```diff
@@ -506,6 +506,9 @@
     <message name="IDS_IDC_BREAK_TILE" desc="The label of the 'Unsplit tabs' tab context menu item. This item exits split view and restore bounded tabs to separate state.">
       Unsplit Tabs
     </message>
+    <message name="IDS_IDC_SWAP_SPLIT_VIEW" desc="The label of the 'Swap split view' tab context menu item">
+      Swap Tab Position
```

Comment:
The attached github issue design has the words "Swap tab positions" (plural). Should this be different?

Reply:
Nice catch! Updated.